auth/service.go

```diff
@@ -1,14 +1,16 @@
 // Copyright (c) Mainflux
 // SPDX-License-Identifier: Apache-2.0
 
-package authn
+package auth
 
 import (
 	"context"
 	"time"
 
 	"github.com/mainflux/mainflux"
+	"github.com/mainflux/mainflux/internal/groups"
```

Comment:
Since Groups are now in the Auth service (we'll have generic groups), there is no need to put this in `internal`, since it won't be shared with other services. We can simply move it to `auth` package.

Reply:
not sure about that since some endpoints may be in other services

Reply:
Yes, some endpoints probably will be in other services, but we won't share them; we won't use the `groups` package as an internally shared package, but as a dependency from the `auth` service. That is, we won't have multiple implementations of the Groups.